packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts

```diff
@@ -197,15 +199,38 @@ export async function deploy(
   ctx.logProvider.info(getLocalizedString("core.deploy.startNotice", PluginDisplayName.Solution));
   const result = await executeConcurrently(thunks, ctx.logProvider);
 
+  const botTroubleShootLink =
```

Comment:
It looks not easy to maintain bot code. Is it possible to have a method to formalize message for Bot, so that other logic keep the same. e.g.
```
const botMsg: string = getBotTroubleShootMsg(); <= if it is not bot project, the value is empty string;
```
Then in L208, it can simply be 
```
const msg = getLocalizedString("core.deploy.successNotice", ctx.projectSetting.appName) + botMsg;
```
Then you can use if !botMsg to decide if it is a bot project and add additional button.

Reply:
Good idea, but not that easy. Since the message displayed in Teams Toolkit is slightly different from the one in output channel, I mean, in logging, we should print `Learn more: https://xxx`, but in UI we don't have to.
It's not just about the flag of isBotProject, but also the output channel of messages.

Reply:
Thanks for pointing this out so that reminding me to do a bit of refactor to improve readability.